### DISCUSSION TITLE
Small display issues.

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#2074] [Ideas]
Hi Seerge,

There is a small defect in the text display in languages that have longer words. 
This is more visible if a limit is applied to the CPU.

Without CPU limit applied:
![03](https://github.com/seerge/g-helper/assets/142881210/0676f31e-90b6-419f-95c6-dd696553a658)

With a CPU limit applied the temperature disappears 
![01](https://github.com/seerge/g-helper/assets/142881210/dd10e8ce-a5ed-418e-aa68-83e18da5890a)
![02](https://github.com/seerge/g-helper/assets/142881210/7e1a71eb-a6fd-4fc4-8517-8eb7f2756cb7)

Perhaps it would be possible to display the temperature and fan speed on a 2nd line just below (if you have space in the form) ? Or reduce the font size?

Something like this:
![Sans titre-1](https://github.com/seerge/g-helper/assets/142881210/3a5107da-e8d1-41c0-a758-ea592d243178)

Could you also increase the number of characters for "144hz+Overdrive" to see "overdrive" in full please.. ;-)

**Comment by seerge** [comment]
@Kikimahe this is a side effect of an old French translation. In English it's just "Mode" which is enough.

I have did a change in Crowdin and merged it into main

Check this build to verify
[GHelper.zip](https://github.com/seerge/g-helper/files/14319226/GHelper.zip)

**Comment by Kikimahe** [comment]
Damn, I didn't think about Crowdin.

It's so much better this way ! 👍 
Thank you very much Seerge. :)

There are just a few pixels missing on the word "Overdrive". 
But I'm going to go modify Crowdin, nothing serious.

**Reply by seerge** [reply]
@Kikimahe sure, it will automatically merge and be in next release :)
